### ISSUE TITLE
Identify the value of integrating various sources

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/backend#394]
We currently have base & core powering the find system. We should now try and vastly expand that. However, it's valuable to ask in what order. The ones currently on our radar as listed in separate issues (under #167 issue / epic). The most important of those are: 

- [x] #68 
- [x] #70 
- [x] #109 
- [x] #111 
- [x] #114 
- [x] #196 
- [x] #203 
- [x] #69 
- [x] #108 
- [x] #110 
- [x] #112 
- [x] #197 
- [x] #303 (the pre-print bits we may not already do)

The critical question here is if there is enough differentiation between these services to make it worth integrating with them - and what would the impact on our users be? I.e what can we say we've achieved at the end of the next sprint. 

Many people use base, probably because it's good & easy, but surely it can't be everything (considering these other services exist). I've done work before (see #181) where I've compared these services in depth - that suggested that there was significant differences (at least within pre-print aggregation) between a handful of the services who're well known above. 

For each of the above, it would be useful to:
* see if they have an API
* get a spreadsheet of all the sources they pull from
* understand their relative size of useful documents (this means papers that are being self-archived, not publishing from oa publishers, in this case)
* flag any clear redundancy (for example if one of these pulls data from another)

@meganwaps, this calls for a huge spreadsheet!

Important to keep in mind, with all of the above, is that the end game here is to get users access to a paper that is currently behind a paywall.

**Comment by meganwaps** [comment]
Bring it on!

**Comment by JosephMcArthur** [comment]
Some notes on this. I made a sheet for base vs openaire. I couldn't do a full evaluation but found out a bunch : https://docs.google.com/spreadsheets/d/178LKl-HTFYt_z7WKFn4kz6Wjb3pzPxJ4BikHBOUI5tA/edit#gid=1232275806. It did seem like there were _some_ differences, but not many. I've suggested we push ahead with having them both. Share had some that base didn't too. However Zenodo & others we don't need with those in. 

In other issues I've made significant progress and signalled what needs to be done next. We can just keep rolling with those as smaller issues as required now.
